### ISSUE TITLE
JPA vs JDBC

### ISSUE THREAD

**Issue by hackjutsu** [BitTigerInst/BitTiger-CS504-FAQ#21]
http://stackoverflow.com/q/11881548/3697757

**Comment by bittiger-java** [comment]
I don't agree JPA is a standard for Object Relational Mapping. JPA is doing more than that. Like typed query in JPQL. Also "Mostly, XML mapping files or annotations on getters, setters can be used to tell the JPA provider which fields on your object map to which fields in the DB" is not accurate. Most of ORM is done by annotation now.
